inc/dps/private/cbor.h

```diff
@@ -208,14 +213,7 @@ DPS_Status CBOR_EncodeInt(DPS_TxBuffer* buffer, int64_t i);
 DPS_Status CBOR_EncodeBytes(DPS_TxBuffer* buffer, const uint8_t* data, size_t len);
 
 /**
- * Encoded a text string
- *
- * @note This function automatically appends the trailing NUL. To encode a string
- * without the trailing NUL use
- * @code
- * CBOR_EncodeLength(buf, strlen(str), CBOR_STRING);
- * CBOR_Copy(buf, str, strlen(str));
- * @endcode
+ * Encode a text string
```

Comment:
I am thinking we should add an API to take a string and a length. This will allow applications to explicitly include NUL characters in the string is that is required.

Reply:
I'll make a note of that for future work.